addons/website/static/src/js/backend/button.js

```diff
@@ -7,46 +7,7 @@ var field_registry = require('web.field_registry');
 
 var _t = core._t;
 
-var WebsitePublishButton = AbstractField.extend({
-    className: 'o_stat_info',
-    supportedFieldTypes: ['boolean'],
-
-    //--------------------------------------------------------------------------
-    // Public
-    //--------------------------------------------------------------------------
-
-    /**
-     * A boolean field is always set since false is a valid value.
-     *
-     * @override
-     */
-    isSet: function () {
-        return true;
-    },
-
-    //--------------------------------------------------------------------------
-    // Private
-    //--------------------------------------------------------------------------
-
-    /**
-     * This widget is supposed to be used inside a stat button and, as such, is
-     * rendered the same way in edit and readonly mode.
-     *
-     * @override
-     * @private
-     */
-    _render: function () {
-        this.$el.empty();
-        var text = this.value ? _t("Published") : _t("Unpublished");
-        var hover = this.value ? _t("Unpublish") : _t("Publish");
-        var valColor = this.value ? 'text-success' : 'text-danger';
-        var hoverColor = this.value ? 'text-danger' : 'text-success';
-        var $val = $('<span>').addClass('o_stat_text o_not_hover ' + valColor).text(text);
-        var $hover = $('<span>').addClass('o_stat_text o_hover ' + hoverColor).text(hover);
-        this.$el.append($val).append($hover);
-    },
-});
-
+// TODO: Remove me when Partner view is adapted to OWL
```

Comment:
> Unfortunately I wasn't able to remove the legacy website_redirect_button as there's still at least one view that needs it. (Invoice -> Create a new Invoice -> Select a partner -> Expand Partner view -> Legacy view with a website redirect button if website_partner is installed.)

Your comment says "remove me", I guess it's more "adapt me", right? Maybe you can already do the adaptation in the same file and keep the old and rename it as `website_redirect_button_legacy` (and use that one in problematic non converted view?) Maybe even move its whole code in the problematic app if it's only one place?